task_DEV_3/task_DEV_3/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+
+namespace task_DEV_3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter the length of the array :");
+            int arrayLength = Convert.ToInt32(Console.ReadLine());
+
+            if (arrayLength <= 5)
+            {
+                int[] numberArray = new int[arrayLength];
+
```

Comment:
It would be better to remove this empty line. It is not mistake but it can improve readability.